src/PhpSpreadsheet/Reader/Csv/Delimiter.php

```diff
@@ -60,7 +60,7 @@ protected function countPotentialDelimiters(): void
 
     protected function countDelimiterValues(string $line, array $delimiterKeys): void
     {
-        $splitString = str_split($line, 1);
+        $splitString = mb_str_split($line, 1);
         if (is_array($splitString)) {
```

Comment:
Unlike str_split, mb_str_split is documented to always return an array. So the following is_array check is now unneeded. Scrutinizer complains about its continued presence, and I am frankly surprised that Phpstan doesn't do likewise. Can you please delete the if statement and its closing bracket, leaving the rest as-is.

Reply:
Thanks, @oleibman 
I have added extra tests and removed the needed condition.